src/app/campaign-stats.model.ts

```diff
@@ -9,8 +9,4 @@ export class CampaignStats {
          */
         public totalCampaignCount: number
     ) {}
-
-    formattedTotalRaised = () => ("£" + this.totalRaised.toLocaleString('en-GB'));
-
-    formattedTotalCount = () => this.totalCampaignCount.toLocaleString('en-GB');
```

Comment:
makes sense we can't put functions on here that we want to call because this class isn't actually used at runtime at all, only used in the type checking stage.